Sources/StreamVideo/Controllers/CallController.swift

```diff
@@ -679,50 +682,127 @@ class CallController: @unchecked Sendable {
             self?.collectAndSendStats()
         }
     }
-    
+
     private func collectAndSendStats() {
         Task {
             do {
-                let stats = try await webRTCClient?.collectStats()
+                let stats = try await webRTCClient.collectStats()
                 await call?.state.update(statsReport: stats)
-                try await webRTCClient?.sendStats(report: stats)
+                try await webRTCClient.sendStats(report: stats)
             } catch {
                 log.error(error)
             }
         }
     }
+
+    fileprivate func _joinWebRTCClient(
```

Comment:
I don't think we should use _ for the private methods, what's the reasoning behind it?

Reply:
Those methods will be removed during the reconnect implementation. They have the underscore to help me (us) remember that they require special handling.